ipaserver/install/server/replicainstall.py

```diff
@@ -1235,7 +1235,11 @@ def promote_check(installer):
 
     except errors.ACIError:
         root_logger.debug(traceback.format_exc())
-        raise ScriptError("\nInsufficient privileges to promote the server.")
+        raise ScriptError("\nInsufficient privileges to promote the server."
+                          "\nPossible issues:"
+                          "\n- An user has insufficient privileges"
```

Comment:
"A user" - you use the article depending on the pronunciation of the following word.